src/aunit/TestRunner.h

```diff
@@ -157,6 +158,18 @@ class TestRunner {
     /** Constructor. */
     TestRunner() {}
 
+    int ellapsedSeconds() const
+    {
+      auto now = std::chrono::high_resolution_clock::now();
+      return std::chrono::duration_cast<std::chrono::seconds>(now - mStartTime).count();
```

Comment:
We cannot do this, because AUnit is designed to be run on the microcontroller itself, and `std::chrono` is not available on AVR compilers. I want avoid the C++ for many reasons, this is one of them.

The problem is that you are overriding the Arduino API `millis()`. In my opinion, it is the wrong place to do your dependency injection. I think `millis()` should be injected at the application layer, not the framework layer.

Reply:
You're right. This is what PR's are made for.
So feel free to dismiss this one :-)